app/serializers/concept_serializer.rb

```diff
@@ -0,0 +1,24 @@
+class ConceptSerializer < ActiveModel::Serializer
+  attributes :id, :name, :uid, :parent_id, :level, :parent_uid
+  delegate :cache_key, :to => :object
+
+  def to_json(*args)
+    Rails.cache.fetch expand_cache_key(self.class.to_s.underscore, cache_key, 'to-json') do
```

Comment:
@kriskelly im not sure how urgent the 'response time' issue is, but perhaps we could also cache at the level of the entire query? ie if no concept has been added, deleted, or updated, then use cache for the whole response